src/kuzya.cpp

```diff
@@ -1530,3 +1554,50 @@ void Kuzya::setAllIconsVisibleInMenu(bool isVisible)
 
 
 }
+
+void Kuzya::writeTemporaryFileState()
+{
+    QSettings* settings = new QSettings(QApplication::applicationDirPath()+"/settings.ini", QSettings::IniFormat);
```

Comment:
I would like to use the same ini file. But according to your last comment i am not allowed to use OptionDialog class. So, i tried to work with this file without OptionDialog class. Am i wrong?

> We should stick to single responsibility principle (see SOLID for reference). Class called OptionsDialog - that means that it is dialog that contains UI for Kuzya options. That is its only responsibility. It should not take care of loading temporary files, dealing with main window configuration, etc. This class should serve only one purpose - manage UI for setting Kuzya options.